yardim-istegi-formlari/enkaz.html

```diff
@@ -83,6 +83,21 @@
                 </a>
                 <h1>Depremzede Bildirim Formu</h1>
                 <div class="form">
+                    <div style="background-color: #2C2E3C4D;  padding: 10px; border-radius: 10px;">
+                        <div style="display: inline-flex; align-items: center; justify-content: space-between;">
+                          <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="rgba(255, 181, 70, 1)" style="width: 30px; height: 30px;"> 
```

Comment:
mobilde svg icon flex wrapperdan dolayı çok küçülüyor 

flex-shrink:0; ile çözülebilir.

aynı durum diğer sayfalardaki iconlarda da mevcut.

![Ekran Resmi 2023-02-09 21 01 07](https://user-images.githubusercontent.com/23366853/217898945-7eab9490-e7e6-4dbf-9f63-d6c339f28a88.png)

Reply:
Bir de ek olarak inline css yazmayalım, dark mode şu an için yok ama eklendiğinde uğraştırır. Css'leri class ile vs yaparsak daha iyi olur.